help/domains/bio/bioinformatics.md

```diff
@@ -0,0 +1,406 @@
+
+### Compositional analysis
+
+Allows you to generate a "weblogo" representation, also known as a
+[sequence logo](https://en.wikipedia.org/wiki/Sequence_logo).
+This visual representation illustrates the letter composition for each position in a sequence.
+
+Typically, a sequence logo is created from a set of aligned sequences
+and helps to identify patterns and variations within the sequences.
+For example, it is commonly used to visualize protein-binding sites in DNA or functional motives in proteins.
+
+![WebLogo representation of sequence composition](img/WebLogo.png)
+
+WebLogo dynamically reflects the sequence sets filtering and allows the
+user to select a subsequence by choosing the residue at a specified position. The tooltip displays the number of
+sequences with a specific monomer at a particular position
+
+![Logo plot](../../uploads/macromolecules/macromolecules-10.gif "Logo plot")
+
+<!---
+
+Have not been tested by me
+
+Another way to perform the composition analysis for MSA data is to use composition Bar charts as any plot in the
+Datagrok reflects all user actions as filtering and selection.
+
+![Bar chart plot](../../uploads/macromolecules/macromolecules-bar-chart-800.gif "Bar chart")
+
+-->
+
+### Similarity search
+
+A linked dataframe displaying the most similar sequences to a selected sequence from the initial table.
+You have the flexibility to adjust the number of sequences displayed and set a similarity cutoff
+in the dataframe properties.
+
+### Diversity search
+
+Creates a new dataframe containing the most diverse sequences within a given dataframe.
+Unlike the [similarity search] feature, this dataframe is not interactive
+and does not respond to sequence selections in the initial dataframe.
+
+### Sequence space
+
+The **Sequence Space** feature in Datagrok is a powerful tool for investigating
+sequence similarity and diversity.
+This is a complex analysis including several steps:
+
+* Calculating sequence similarity
+* Performing [dimensionality reduction](https://en.wikipedia.org/wiki/Dimensionality_reduction)
+to transformation sequence similarity into 2D space,
+* Visualizing the results on a scatter plot
+
+As a result, the sequences are projected to
+2D space closer if they correspond to similar structures, and farther otherwise.
+
+To launch the analysis from the top menu, select _Bio | Sequence space_
+and specify the column that contains the sequences of interest.
+
+![Sequence Space parameters](img/Sequence_Space_parameters.png)
+
+Sequence Space supports various metrics for measuring sequence similarity,
+such as Tanimoto and Sokal, and offers two methods of dimensionality reduction: UMAP and t-SNE.
+
+By using Sequence Space, you can uncover clusters of similar sequences,
+facilitating the identification of groups with shared characteristics.
+For example, it can help to identify clusters of antibodies with similar CDRs
+(complementarity-determining regions).
```

Comment:
For example, it can help to identify clusters of antibodies with similar CDRs
(complementarity-determining regions). ->
It is mostly used to separate the groups of sequences with common motifs like complementarity-determining regions for antibodies